compliance_applications.go

```diff
@@ -108,6 +108,9 @@ type ListComplianceApplicationResponse struct {
 
 func (service *ComplianceApplicationService) Get(complianceApplicationId string) (response *ComplianceApplicationResponse, err error) {
 	req, err := service.client.NewRequest("GET", nil, "ComplianceApplication/%s", complianceApplicationId)
+	if err != nil {
```

Comment:
you can choose to ignore the error above, in order to maintain the existing code logic
i.e. `req, _ := service.client.NewRequest()`

similar thing at other places as well

Reply:
If something wrong in NewRequest, will return error https://github.com/plivo/plivo-go/blob/master/baseclient.go#L41
Not good to ignore the err.